src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.OSX/ChainPal.cs

```diff
@@ -413,8 +413,8 @@ private static X509ChainStatusFlags FindUntrustedRootReason(X509Certificate2 cer
 
         private X509ChainElement BuildElement(X509Certificate2 cert, int dwStatus)
         {
-            const int errSecCertificateExpired = -67818;
-            const int errSecCertificateNotValidYet = -67819;
+            const int ErrSecCertificateExpired = -67818;
+            const int ErrSecCertificateNotValidYet = -67819;
```

Comment:
Technically these should be camelCase, since they're matching the OS constant casing.

Reply:
Technically they should live in Interop in that case :)

I can change back.